src/PDFontMetrics.jl

```diff
@@ -161,7 +161,7 @@ function get_font_widths(cosdoc::CosDoc, font::CosObject)
     widths = get(widths_obj, true)
     for i = firstchar:lastchar
         ix = i - firstchar + 1
-        d[i+1] = widths[ix]
+        d[i+1] = round(Int,widths[ix])
```

Comment:
There may be several changes needed for the `pdDocGetOutlines` function. I will suggest #44 be fixed in a separate check-in.

Reply:
Separate PR submitted: #46